### PR TITLE
Show current as Amperes in OSD

### PR DESCRIPTION
**Pull request by sixtyfive**
In configurator and firmware:

- change adc_c to adc2_c everywhere
- introduce 2nd coefficient for ADC1
- submit ADC1/2 raw values over USB
- display ADC1/2 raw values in configurator

Also adds ignores for data files in configurator directory.

### CODE REVIEW

File: configurator/main.c [hunk 1]
```diff
@@ -8,8 +8,7 @@
 #include "usb.h"
 
 void get_all(void) {
-  int value;
-  printf("            TUNING\n");
```
Comment (catphish) [inline configurator/main.c:12]: What did this heading text do to offend you?

**Comment by catphish** [comment]
This looks good apart from accidentally deleting the TUNING header.
I would actually like to report a lot more raw values over USB as this will eventually allow a configurator GUI to display things like pitch/roll for testing.

File: configurator/main.c (hunk 1 above)
Comment (sixtyfive) [reply configurator/main.c:12]: argl, nothing. that's VSCode's "great" staging of "single" lines.

**Comment by sixtyfive** [comment]
> This looks good apart from accidentally deleting the TUNING header. I would actually like to report a lot more raw values over USB as this will eventually allow a configurator GUI to display things like pitch/roll for testing.

That sounds lovely :)

**Pull request merged**